pkg/apis/kops/validation.go

```diff
@@ -377,6 +378,42 @@ func (c *Cluster) Validate(strict bool) error {
 		}
 	}
 
+	// Blacklisted Values
+	err = c.isValidAgainstBlacklist()
+	if err != nil {
+		return fmt.Errorf("Unable to validate against blacklist: %v", err)
+	}
+
+	return nil
+}
+
+type ClusterSpecBlackList struct {
+	KubernetesVersions []string
+}
+
+// Todo Kris - We need to get this into a yaml file that can be read in at compile time.. or do we?
+// If we decide to put this in a model of sorts, we will need to generate the bin data for it..
+// kind of sounds like we are walking down the model path again - so trying things out like this for now
+var CSBlacklist = ClusterSpecBlackList{KubernetesVersions: []string{"1.5.0", "v1.5.0"}}
+
+
+// Will validate our cluster against the blacklist
+func (c *Cluster) isValidAgainstBlacklist() (error) {
+
+	isb := os.Getenv("KOPS_BYPASS_BLACKLIST")
+	if isb != "" {
+		// They are sure they want to bypass the validation check
+		return nil
+	}
+
+	// --------------------------------------------------------------------------------
+	// Kubernetes Version
+	//
+	for _, blacklistVersion := range CSBlacklist.KubernetesVersions {
+		if c.Spec.KubernetesVersion == blacklistVersion {
+			return fmt.Errorf("blacklisted kubernetes version %s detected. Bypass by setting $KOPS_BYPASS_BLACKLIST=1", blacklistVersion)
```

Comment:
I think it would be `export KOPS_BYPASS_BLACKLIST=1`, right?  I'm not familiar with the `$` syntax?